### PR TITLE
test(compute): add headers test

### PR DESCRIPTION
**Pull request by georgiyekkert**
This test is a requirement by vchudnov

### CODE REVIEW

File: compute/apiv1/smoke_test.go [hunk 1]
```diff
@@ -593,3 +598,35 @@ func TestCapitalLetter(t *testing.T) {
 		t.Fatalf("got(-),want(+):\n%s", diff)
 	}
 }
+
+func TestHeaders(t *testing.T) {
+	if testing.Short() {
```
Comment (codyoss) [inline compute/apiv1/smoke_test.go:603]: nit: you can remove the skip. This test does not hit any backend resources so it should be fine to run all the time.

**Review by codyoss — CHANGES_REQUESTED** [review]
Just one minor thing

File: compute/apiv1/smoke_test.go (hunk 1 above)
Comment (noahdietz) [reply compute/apiv1/smoke_test.go:603]: I think the client factory method will try to load ADC & fail if not present, so if this test is using an insecure endpoint we may want to also pass `WithoutAuthentication` [option](https://pkg.go.dev/google.golang.org/api/option#WithoutAuthentication).

Comment (georgiyekkert) [reply compute/apiv1/smoke_test.go]: Done

**Review by codyoss — APPROVED** [review]
Thanks!

**Pull request merged**